lib/python/treadmill_aws/cli/admin/aws/instance.py

```diff
@@ -122,10 +122,20 @@ def configure(instance):
         '--hostname',
         help='Shortname or Pattern, e.g. PATTERN-{time}',
     )
+    @click.option(
+        '--ip-address',
+        required=False,
+        help='IP address',
+    )
+    @click.option(
+        '--eni',
+        required=False,
+        help='Network id',
```

Comment:
Can you give an example of the format you expect the command to be in? 'Network ID' is vague.
e.g. eni-xxxxxxxx

Reply:
treadmill --dns-domain DOMAIN admin aws instance create --image AMI_ID --eni ENI_ID --size INSTANCE_TYPE --count 1 --role ROLE --hostname HOSTNAME

Reply:
That's not what I meant. "ENI" is an acronym that not everyone may recognize immediately. 
In the **--help=** text, please give an example of the format you expect the user to input. 
e.g. help='Elastic Network ID; e.g. eni-xxxxxxxx' gives a better indication of what the flag does and what we expect for input.

Reply:
totally agree. The change has been made.